Backpack/src/main/java/net/skyscanner/backpack/text/BpkText.kt

```diff
@@ -76,6 +77,7 @@ open class BpkText @JvmOverloads constructor(
     }
 
   private var _textColour: ColorStateList? = null
+  private var _textColourLink: ColorStateList? = null
```

Comment:
nit: this should be `color` not `colour`. We used American english in code.

Reply:
Done